packages/documentation/src/pages/getting-started/index.mdx

```diff
@@ -54,43 +60,45 @@ git clone git@github.com:department-of-veterans-affairs/vets-api.git
-  - Mock data used when running locally and on dev for the backend (must be a sibling of vets-api)
+*Front end repos*
+- **vets-website**: Core front end platform and application code
+- **vagov-content**: Markdown content used to generate static pages
+- **veteran-facing-services-tools**: Shared front end components (including non VA.gov users) and front end documentation site
 
-## Start up the front end
+*Back end repos*
+- **vets-api**: Core Rails API server application code
+- **vets-api-mockdata**: Mock data used when running locally and on dev for the backend
 
-Once you have Node and Yarn installed, you can install the dependencies for vets-website:
+*Shared repos*
+- **vets-json-schema**: Shared JSON Schema definitions used by form applications and the APIs that they consume
 
+## Start up the front end
+
+7. Install `vets-website` dependencies
 ```bash
 yarn install
 ```
 
-Once this completes, you should be able to start up the site locally:
-
+8. Start the local development server
 ```bash
 yarn watch
 ```
 
-Once this builds (you'll see a message about Webpack compilation finishing), open up [http://localhost:3001](http://localhost:3001) and you should see the VA.gov homepage.
+*Build is complete when the CLI says* `Compiled successfully`
+
+9. Open the [http://localhost:3001](http://localhost:3001) in a browser
+
+## Backend and internal tools set up
+
+Setting up the backend and internal tools allows local test account login and static content rendering.
+
+- **Backend set up** instructions: [vets-api](https://github.com/department-of-veterans-affairs/vets-api).
 
-## Set up the backend
+*Follow the Docker instructions, which are in the README for the `vets-api` repo.*
 
-You can find the backend set up instructions in the [vets-api](https://github.com/department-of-veterans-affairs/vets-api) repo. If possible, you should follow the Docker instructions, which are in the README for that repo.
+- **Local test account login** instructions: [Accessing Staging](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Work%20Practices/Accessing-Staging.md)
 
-Once you've gone through that set up, you should verify that you can log in using one of the test accounts, which can be found in the [Accessing Staging](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Work%20Practices/Accessing-Staging.md) doc.
 
-## Internal tools
+- **Internal tools setup** instructions: [Accessing internal tools](internal-tools).
 
-Once you've got the site running locally, you'll also want to get set up with our [internal tools](internal-tools). This will enable you to render static pages that use Drupal data locally, as well as access our reporting and monitoring tools. 
+*This proxy setup is required to access static content locally and to access to our reporting and monitoring tools. Running the watch task with the SOCK proxy active will automatically pull and cache the static content for `vets-website`*
```

Comment:
We could add a section here for additional info. It could link to a summary of all of our repos and decision documentation like "Why we use yarn and not npm"